src/js/tabs/setup.js

```diff
@@ -369,29 +370,39 @@ setup.initialize = function (callback) {
 
             if (supported && ispConnected()) {
                 const buildRoot = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
+
                 const buildConfig = `<span class="buildInfoBtn" title="${i18n.getMessage(
                     "initialSetupInfoBuildConfig",
                 )}: ${buildRoot}/json">
-                                    <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage(
+                    <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage(
     "initialSetupInfoBuildConfig",
 )}</strong></a></span>`;
+
                 const buildLog = `<span class="buildInfoBtn" title="${i18n.getMessage(
                     "initialSetupInfoBuildLog",
                 )}: ${buildRoot}/log">
-                                    <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage(
+                    <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage(
     "initialSetupInfoBuildLog",
 )}</strong></a></span>`;
-                build_info_e.html(`<span class="buildInfoBtn" title="${i18n.getMessage(
-                    "initialSetupInfoBuildOptions",
-                )}">
-                    <a class="buildOptions" href=#"><strong>${i18n.getMessage(
-        "initialSetupInfoBuildOptionList",
-    )}</strong></a></span>`);
+
                 build_info_e.html(`${buildConfig} ${buildLog}`);
+
+                const buildDownload = `<span class="buildInfoBtn" title="${i18n.getMessage(
+                    "initialSetupInfoBuildDownload",
+                )}: ${buildRoot}/hex">
+                    <a href="${buildRoot}/hex" target="_blank"><strong>${i18n.getMessage(
+    "initialSetupInfoBuildDownload",
+)}</strong></a></span>`;
+
+                build_firmware_e.html(buildDownload);
             } else {
                 build_info_e.html(
                     supported ? i18n.getMessage("initialSetupNotOnline") : i18n.getMessage("initialSetupNoBuildInfo"),
                 );
+
+                build_firmware_e.html(
+                    supported ? i18n.getMessage("initialSetupNotOnline") : i18n.getMessage("initialSetupNoBuildInfo"),
```

Comment:
There's a bit of code and UI duplication here. The UI duplication seemed fine and I didn't know what else to put. The code duplication could be refactored to assign and reuse the ternary instead of duplicating it. This seemed clearer at no real cost.